modules/php/Game/Utils.php

```diff
@@ -183,4 +184,99 @@ public static function checkForPoorBonus($player_id)
       Utils::getGame()->performDrawCards($player_id, $poorBonus);
     }
   }
+
+  public static function playerHasNotYetUsedGoalMill()
```

Comment:
All those functions are only used in their respective class. Would it make sense to simply move them there instead of in the game utils?

Reply:
Same reasoning as with the "NotYet", just to have consistency for all the "playerTurn" game state value checks.
And probably you might have to check them in other places for improved UI handling?